UserInterface/GameManager/GameManager.cs

```diff
@@ -67,45 +88,80 @@ private void RunGame()
             while (isGameOnGoing)
             {
                 Console.CursorVisible = false;
-                Pair? currentPair = _pairManager.Pair;
 
-                _board.FillBoardWithAnimals(_animals);
+                _board.FillBoardWithItems(_gameItems);
 
                 _userInterface.DrawBoard(_board);
-                _board.RemoveAnimalFromBoard(_animals);
+                _board.PrepareBoard(_gameItems);
+                
+                _movementManager.Act(_gameItems, _board);
 
-                _animalActions.Move(_board, _animals);
+                _pairManager.RemoveNotExistingPairs(_pairs);
+               
+                _pairManager.CheckPairForExistence(_pairs, _board, _gameItems, _childrenGameItems);
 
-                _pairManager.RemoveNonExistsPairs(_pairs);
+                _pairManager.AddPairToList(_pairManager.CreatePair(_gameItems), _pairs);
+                
+                _itemManager.ProcessChildrenItems(_gameItems, _childrenGameItems);
 
-                _pairManager.CheckPairForExistence(_pairs, _board, _animals, _children);
+                _itemManager.RemoveInactiveItems(_gameItems);
+                Thread.Sleep(ConstantsRepository.ThreadSleep);
+                ConsoleKey? consoleKey = _userInterface.GetInputKey();
+            
+                foreach (var info in _gameItemsInfo)
```

Comment:
Code covered by this foreach is related to _itemManager, so it should be implemented there.
RunGame should use only functionality of _itemManager not do the job instead of it.

Reply:
Done.